changelog-entries/203.md

```diff
@@ -0,0 +1 @@
+- Fixed the out-of-bounds data access which might cause trouble in 2D parallel cases [#202](https://github.com/precice/openfoam-adapter/issues/202)
```

Comment:
I am not sure if the user can clearly understand what this means without looking at the code. Maybe, instead:

```suggestion
- Fixed a potential memory access issue in the xy-alignment check for 2D cases [#202](https://github.com/precice/openfoam-adapter/issues/202)
```